lib/Runtime/PlatformAgnostic/Platform/Unix/AssemblyCommon.cpp

```diff
@@ -0,0 +1,32 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "Common.h"
+#include "PlatformAgnostic/AssemblyCommon.h"
+
+extern void mac_fde_wrapper(const char *dataStart, mac_fde_reg_op reg_op)
+{
+    for(const char *head = dataStart, *last = dataStart + CC_FDE_SIZE;
+        head != last;
+        /* noop */)
+    {
+        const char *op = head;
+        // Get Length of record [ Read 4 bytes ]
+        const uint32_t* op_data = (const uint32_t*)op;
```

Comment:
I meant the pointer `op_data` was not as straightforward. Just do `const uint32_t length = *(const uint32_t*)op;` and use `length`.